druid-shell/src/window.rs

```diff
@@ -253,14 +229,11 @@ impl WindowHandle {
         self.0.bring_to_front_and_focus()
     }
 
-    /// Request that [`prepare_paint`] and [`paint`] next time there's the opportunity
-    /// to render another frame. This differs from [`invalidate`] and [`invalidate_rect`]
-    /// in that it doesn't invalidate any part of the window.
-    ///
-    /// [`prepare_paint`]: trait.WinHandler.html#tymethod.prepare_paint
-    /// [`paint`]: trait.WinHandler.html#tymethod.paint
-    /// [`invalidate`]: struct.WindowHandle.html#tymethod.invalidate
-    /// [`invalidate_rect`]: struct.WindowHandle.html#tymethod.invalidate_rect
+    /// Request that [`prepare_paint`](WinHandler::prepare_paint) and [`paint`](WinHandler::paint)
+    /// be called next time there's the opportunity to render another frame. This differs from
+    /// [`invalidate`](WindowHandle::invalidate) and
+    /// [`invalidate_rect`](WindowHandle::invalidate_rect) in that it doesn't invalidate any part
+    /// of the window.
```

Comment:
my inclination would be to continue keeping the links at the bottom, to make it easier to read in the code itself; this still works with intradoc links.